infrastructure-playbooks/vars/lv_vars.yaml

```diff
@@ -0,0 +1,48 @@
+# This file configures logical volume creation for FS Journals on NVMe, a NVMe based bucket index, and HDD based OSDs.
+# This playbook configures one NVMe device at a time. If your OSD systems contain multiple NVMe devices, you will need to edit the key variables ("nvme_device", "hdd_devices") for each run.
+# It is meant to be used when osd_objectstore=filestore and it outputs the the necessary input for group_vars/osds.yml when configured with osd_scenario=lvm
+# The LVs for journals are created first then the LVs for data. All LVs for journals correspond to a LV for data
+#
+## CHANGE THESE VARS ##
+
+# Path of nvme device primed for LV creation for journals and data. Only one NVMe device is allowed at a time.
+nvme_device: /dev/nvme0n1
```

Comment:
What if someone passes a list? We need a safety check that fails if length is > 1

Reply:
when `nvme_device` has more than one entry like

`nvme_device:`
  `- xx`
  `- yy`

using length of `nvme_device` works, but when `nvme_device: /dev/nvme0n1 `, length returns the length of the variable. 
How can I ensure that `nvme_device:` is one value and not a list?

Reply:
can we try and solve this one at a later iteration? I don't think this should prevent the PR from moving forward

Reply:
@alfredodeza sure, I can clarify it in the comments for now.